components/pool-pair-card/index.tsx

```diff
@@ -31,22 +32,28 @@ const LQDPoolPairCard = ({ pool, navigationVariant = 'primary' }: PoolPairCard)
   const change = pool.apr;
 
   return (
-    <TouchableOpacity onPress={handlePress} style={styles.container}>
-      <View style={styles.topContainer}>
-        <LQDPoolImages tokenAIconURL={tokenAIconURL} tokenBIconURL={tokenBIconURL} />
-
-        <View style={[styles.changeContainer, { backgroundColor: backgroundColors[+increased] }]}>
-          {increased ? (
-            <ArrowUpAltIcon width={12} height={12} fill={textColors[+increased]} />
-          ) : (
-            <ArrowDownIcon width={12} height={12} fill={textColors[+increased]} />
-          )}
-          <Text style={[styles.change, { color: textColors[+increased] }]}>{change}%</Text>
-        </View>
-      </View>
-
-      <Text style={styles.title}>{symbol}</Text>
-    </TouchableOpacity>
+    <>
+      {!loading && (
+        <TouchableOpacity onPress={handlePress} style={styles.container}>
+          <View style={styles.topContainer}>
+            <LQDPoolImages tokenAIconURL={tokenAIconURL} tokenBIconURL={tokenBIconURL} />
+
+            <View style={[styles.changeContainer, { backgroundColor: backgroundColors[+increased] }]}>
+              {increased ? (
+                <ArrowUpAltIcon width={12} height={12} fill={textColors[+increased]} />
+              ) : (
+                <ArrowDownIcon width={12} height={12} fill={textColors[+increased]} />
+              )}
+              <Text style={[styles.change, { color: textColors[+increased] }]}>{change}%</Text>
+            </View>
+          </View>
+
+          <Text style={styles.title}>{symbol}</Text>
+        </TouchableOpacity>
+      )}
+
+      {loading && <LQShrimeLoader style={styles.loader} />}
+    </>
```

Comment:
Can we check for loading first?

```suggestion
if(loading) {
 return  <LQShrimeLoader style={styles.loader} />
}

return (
      <TouchableOpacity onPress={handlePress} style={styles.container}>
          <View style={styles.topContainer}>
            <LQDPoolImages tokenAIconURL={tokenAIconURL} tokenBIconURL={tokenBIconURL} />

            <View style={[styles.changeContainer, { backgroundColor: backgroundColors[+increased] }]}>
              {increased ? (
                <ArrowUpAltIcon width={12} height={12} fill={textColors[+increased]} />
              ) : (
                <ArrowDownIcon width={12} height={12} fill={textColors[+increased]} />
              )}
              <Text style={[styles.change, { color: textColors[+increased] }]}>{change}%</Text>
            </View>
          </View>

           <Text style={styles.title}>{symbol}</Text>
       </TouchableOpacity>
)

```